orkaudio/filters/LiveStream/LiveStreamSession.cpp

```diff
@@ -0,0 +1,125 @@
+/*
+ * Oreka -- A media capture and retrieval platform
+ * 
+ * Copyright (C) 2005, orecx LLC
+ *
+ * http://www.orecx.com
+ *
+ * This program is free software, distributed under the terms of
+ * the GNU General Public License.
+ * Please refer to http://www.gnu.org/copyleft/gpl.html
+ *
+ */
+#pragma warning(disable : 4786) // disables truncated symbols in browse-info warning
+#define _WINSOCKAPI_            // prevents the inclusion of winsock.h
+
+#include "Utils.h"
+#include "AudioCapture.h"
+#include "LiveStreamSession.h"
+#include "AudioCapturePlugin.h"
+#include "AudioCapturePluginCommon.h"
+#include <set>
+
+extern AudioChunkCallBackFunction g_audioChunkCallBack;
+extern CaptureEventCallBackFunction g_captureEventCallBack;
+
+#define CONFERENCE_TRANSFER_TRACKING_TAG_KEY "orig-orkuid"
+
+LiveStreamSessions::LiveStreamSessions()
+{
+    voIpSessions = VoIpSessionsSingleton::instance();
+}
+
+bool LiveStreamSessions::StartStreamNativeCallId(CStdString &nativecallid)
+{
+    std::map<unsigned long long, VoIpSessionRef>::iterator pair;
```

Comment:
not required to define pair before hand. Pair can be an auto field.

Reply:
done